x-pack/plugins/ml/public/application/datavisualizer/file_based/components/file_datavisualizer_view/file_datavisualizer_view.js

```diff
@@ -276,8 +283,12 @@ export class FileDataVisualizerView extends Component {
     return (
       <div>
         {mode === MODE.READ && (
-          <React.Fragment>
-            {!loading && !loaded && <AboutPanel onFilePickerChange={this.onFilePickerChange} />}
+          <>
+            {!loading && !loaded && (
+              <>
```

Comment:
I wonder if this fragment wrapper can be removed since it's only one element on the inside, maybe a leftover from refactoring?